src/Interpreters/InterpreterSelectQuery.cpp

```diff
@@ -810,12 +807,28 @@ void InterpreterSelectQuery::executeImpl(QueryPlan & query_plan, const BlockInpu
     bool intermediate_stage = false;
     bool to_aggregation_stage = false;
     bool from_aggregation_stage = false;
+    const bool filter_in_prewhere = (
```

Comment:
Name is not very good. Logically, this flag tells if if filter actions for row-level security is executed as part of prewhere or as a separate step. Maybe `execute_row_level_filter_in_prewhere` ?

Reply:
Fixed.